common/src/main/java/brocoeur/example/common/cointoss/strategy/HeadOnlyCoinTossBy5Euro.java

```diff
@@ -0,0 +1,19 @@
+package brocoeur.example.common.cointoss.strategy;
+
+import brocoeur.example.common.CoinTossGameStrategy;
+import brocoeur.example.common.Gamble;
+import brocoeur.example.common.cointoss.CoinTossPlay;
+
+import java.util.List;
+
+public class HeadOnlyCoinTossBy5Euro implements CoinTossGameStrategy {
+
+    @Override
+    public Gamble play(int availableMoney, List<CoinTossPlay> previousRoulettePlay) {
+        if (availableMoney >= 5) {
+            return new Gamble(CoinTossPlay.HEAD, 5);
+        }
+        return new Gamble(CoinTossPlay.HEAD, availableMoney);
+    }
+
+}
```

Comment:
We can't do classes for every amount we want to play. Do something more generic like percentage of the gambled amount or something similar

Reply:
This is a specific strategy but yes we can use percentage if needed.